.github/workflows/eden.yml

```diff
@@ -22,7 +22,48 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check_pr_publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install curl and jq
+        run: apt-get update && apt-get install -y curl jq
+
+      - name: Check Image Availability
```

Comment:
It would be good to rename it so something like. `Check for eve pr build tag on docker hub`

Reply:
done.